core/src/main/java/com/graphhopper/routing/weighting/GenericWeighting.java

```diff
@@ -124,40 +119,16 @@ public double calcWeight(EdgeIteratorState edgeState, boolean reverse, int prevO
         if (time == Long.MAX_VALUE)
             return Double.POSITIVE_INFINITY;
 
-        int spatialId = gEncoder.getSpatialId(edgeState.getFlags());
```

Comment:
Did I missed it or is the SpatialIndex actually unused now? We safe the CountryId but don't evaluate it.